network_analysis/service-area-task/README.md

```diff
@@ -1,43 +1,38 @@
-# Service Area Task
+# Service area task
 
-Find the service area for a point.
+Find the service area within a network from a given point.
 
-A service area shows locations that can be reached from a facility based off a certain impedance [such as travel time]. Barriers can also be added which can effect the impedance by not letting traffic through or adding the time is takes to pass that barrier.
+![Image of service area task](ServiceAreaTask.gif)
 
-![](ServiceAreaTask.gif)
+## Use case
 
-## How to use sample
+A service area shows locations that can be reached from a facility based off a certain impedance, such as travel time or distance. Barriers can increase impedance by either adding to the time it takes to pass through the barrier or by altogether preventing passage.
 
-1. In order to find any service areas at least one ServiceAreaFaciltiy needs to be added.
-2. To add a facility, click the facility button, then click anywhere on the MapView.
-3. To add a barrier, click the barrier button, and click multiple locations on MapView.
-    * Hit the barrier button again to finish drawing barrier.
-    * Hitting any other button will also stop the barrier from drawing.
-4. To show service areas around facilities that were added, click show service areas button.
-5. Reset button, clears all graphics and reset ServiceAreaTask.
+You might calculate the region around a hospital in which ambulances can service in 30 min or less.
 
-## How it works
+## How to use the sample
+
+In order to find any service areas at least one facility needs to be added to the map view. Add facilities to the map by clicking the 'Add Facilities' button and then clicking anywhere on the map. To add a barrier, click the 'Add Barrier' button, and click multiple locations on map. Clicking any other button will stop the barrier from drawing. To show the service areas around facilities that were added, click 'Show Service Areas' button. The 'Reset' button clears all graphics and resets the service area task.
```

Comment:
I think we've been learning towards using double quotes for button names

Reply:
Yep, this was inconsistent. Updated!